overview.md

```diff
@@ -33,80 +31,91 @@ search across any set of indexed STAC [`Items`](stac-spec/item-spec/README.md).
-[OpenAPI](http://docs.opengeospatial.org/is/17-069r3/17-069r3.html#_requirements_class_openapi_3_0) options, as STAC 
+[OpenAPI](http://docs.opengeospatial.org/is/17-069r3/17-069r3.html#_requirements_class_openapi_3_0) options, since STAC 
 is always in GeoJSON and OpenAPI is used to specify STAC API. Full compliance involves splitting STAC `Items` into
 individual `/collections/{collectionId}/items` endpoints that expose querying single collections, as OAFeat does
 not currently allow cross-collection search. And it adds a few other requirements, which are highlighted in the 
 [features description](ogcapi-features/), in order to help STAC implementors understand OAFeat without having to
 read the full spec from scratch.
 
-### Extensions
+### Extensions & Fragments
+
+Both STAC API and OAFeat allow 'extensions' that can be added for additional functionality. The STAC community has 
+created a number of extensions to OAFeat, in order to meet requirements of its implementors, and the complete list 
+can be found in the [extensions document](extensions.md), which links to each of them and details their maturity . 
+These are specified in OpenAPI, which works cleanly when the new functionality is a new api location (a complete 
+resource, or adding POST to an existing one). Many of the additions, however, are done at the parameter or response 
+level, like adding a `sortby` field to return ordered results. To make these reusable by both Item Search and OAFeat, 
+in STAC they are specified in the [fragments/](fragments/) directory, where the core functionality is described.
+Each *extension* is made specifically against a part of the STAC API, so that it can be specified as a conformance
+class (to say, for example, that item-search supports `sortby` but ogcapi-features does not). But if it is based
+on a fragment then it will likely just be a thin wrapper to declare the conformance. This is a bit less than ideal,
+but it seemed to be the best approach that actually works with OpenAPI.
+
+We are working to fully align STAC's OAFeat extensions to be the same as the OGC API building blocks being worked on,
+so that every STAC extension is specified at the OGC API level. The end goal is for STAC API to just present a 
+curated set of extension options.
+
+## STAC Core and OGC Versions
 
-Both STAC API and OAFeat allow 'extensions' that can be added for additional functionality.
-We are working to fully merge the extensions, so that every STAC extension is specified at the OGC API level, and
-STAC API just presents a curated set of extension options. 
+This version of STAC API depends on OGC API - Features - Part 1: Core [Version 1.0](http://docs.opengeospatial.org/is/17-069r3/17-069r3.html),
+and on [OGC API - Commons - Part 2: Collections](https://github.com/opengeospatial/ogcapi-common/blob/cc8ca2f011d7e1b19721268c4bf2b97c163d160a/20-024.pdf)
```

Comment:
Remove references to Commons for now and move it to the corresponding PR? Also it seems master is the most-current branch, not August

Reply:
Good catch, will remove it. And yeah, figuring out current branch was a big part of the reason I wanted to remove it. Oh yeah, and I didn't want to just link to master, since master may change.

Reply:
I'm hoping they'll have something better for us to refer to by the time we merge this PR, or at least will update to the latest. So I'll just close here, but will make a comment on that PR.